lib/helpers.js

```diff
@@ -171,11 +171,17 @@ export function getTempDir(prefix = '') {
         return;
       }
 
-      fs.realpath(folder, (realError, rpath) => (realError ? reject(realError) : resolve(rpath)));
+      resolve(realPath(folder));
     });
   });
 }
 
+export function realPath(folder) {
+  return new Promise((resolve, reject) => {
+    fs.realpath(folder, (err, rpath) => (err ? reject(err) : resolve(rpath)));
+  });
+}
+
```

Comment:
Yeah, not opposed to using a library. But would prefer to not tackle it in this PR